source/common/upstream/load_balancer_impl.h

```diff
@@ -228,13 +228,16 @@ class RoundRobinLoadBalancer : public LoadBalancer, ZoneAwareLoadBalancerBase {
   struct Scheduler {
     // EdfScheduler for weighted RR.
     EdfScheduler<const Host> edf_;
-    // Simple clock hand for when we do unweighted.
+    // Simple clock hand for when we do unweighted. Tracks the total number of
+    // picks when weighted.
     size_t rr_index_{};
     bool weighted_{};
   };
 
   // Scheduler for each valid HostsSource.
   std::unordered_map<HostsSource, Scheduler, HostsSourceHash> scheduler_;
+  // Seed to allow us to desynchronize WRR balancers across a fleet.
```

Comment:
Mind adding a few sentences about this here, or other place of your choice?  I think it would be helpful if this were  spelled out more for folks who haven't seen a tech talk (or a postmortem) of what happens in prod recently :-)